force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +140,24 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
+        Integer numberOfVowels = 0;
+        List<string> lettersInWord = new List<String>();
 
-        /*
-        for () {
+        for (Integer i = 0; i < str.length(); i++) {
             // get individual characters from the string - hint use substring or split
-            // check if the character is a vowel
+            lettersInWord.add(str.substring(i, i+1));            
+        }
+        
+        for (String letter : lettersInWord) {
+        // check if the character is a vowel
+            if (letter == 'a' || letter == 'e' || letter == 'i' || letter == 'o' || letter == 'u') {
             // if it is a vowel, increment the result Integer
+            numberOfVowels += 1;
+            }    
```

Comment:
When you are checking for membership. In this case does "is my variable a member of the set of vowels. It is much easier to use a set. Conditional logic is always hard to read and test. 

  Set<String> vowels = new Set<String>{
                'a', 'e', 'i', 'o', 'u'
        };